bin/queuePopulator.js

```diff
@@ -60,6 +62,19 @@ async.waterfall([
         });
         done();
     },
+    done => {
+        healthServer.onReadyCheck(log => {
+            const state = queuePopulator.zkStatus();
+            if (state.code === 3) {
```

Comment:
You should use the symbolic constant exposed by the zookeeper client module (may be `zookeeper.State.SYNC_CONNECTED`?)